be/src/olap/memtable_flush_executor.cpp

```diff
@@ -122,7 +131,8 @@ Status FlushToken::_do_flush_memtable(MemTable* memtable, int32_t segment_id, in
     return Status::OK();
 }
 
-void FlushToken::_flush_memtable(MemTable* memtable, int32_t segment_id, int64_t submit_task_time) {
+void FlushToken::_flush_memtable(MemTable* mem_table, int32_t segment_id,
```

Comment:
renaming to mem_table does not make any sense

Reply:
Just unified the param name and make the clang-tidy happy.

Reply:
> Just unify the param name with declaration and make the clang-tidy happy.

we all know that clang-tidy's happiness is super important, so is memtable's. Maybe renaming the declaration is a win-win solution :P